instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/instrumentation.rb

```diff
@@ -33,8 +33,12 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           defined?(::GraphQL)
         end
 
-        def legacy_tracing_requirement_satisfied?
-          Gem::Requirement.new('<= 2.0.17').satisfied_by?(gem_version) || Gem::Requirement.new('2.0.19').satisfied_by?(gem_version)
+        def supports_legacy_tracer?
+          Gem::Requirement.new('!= 2.0.18').satisfied_by?(gem_version)
```

Comment:
👍 
```
irb(main):003:0> Gem::Requirement.new('!= 2.0.18').satisfied_by?(Gem::Version.new("2.0.18"))
=> false
irb(main):004:0> Gem::Requirement.new('!= 2.0.18').satisfied_by?(Gem::Version.new("2.0.17"))
=> true
irb(main):005:0> Gem::Requirement.new('!= 2.0.18').satisfied_by?(Gem::Version.new("2.0.19"))
=> true
```

Reply:
https://github.com/rmosolgo/graphql-ruby/issues/4378 
https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/374

introduced in 2.0.18, [fixed in 2.0.19](https://github.com/rmosolgo/graphql-ruby/blob/master/CHANGELOG.md#2019-14-march-2023)